website_sale_require_legal/views/website_sale.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<template id="checkout" inherit_id="website_sale.checkout">
+    <xpath expr="//div[@class='js_shipping row mb16']" position="after">
+        <!-- Only shown for public users, out of test mode -->
+        <div t-if="user_id == website.user_id and
+                   'TestCursor' not in repr(env.cr)"
+             class="row">
+            <div t-attf-class="form-group col-md-12
+                               #{error.get('accepted_legal_terms') and
+                                 'has-error' or ''}">
+                <label for="accepted_legal_terms" class="control-label">
+                    <input type="checkbox"
+                           name="accepted_legal_terms"
+                           id="accepted_legal_terms"
+                           required="required"/>
+                    I accept the <a href="/page/legal">legal advice</a>,
```

Comment:
You shouldn't introduce here these break lines and indentation to avoid a weird translation strings